packages/api/src/utils/car.js

```diff
@@ -8,7 +8,7 @@ import { sha256 } from 'multiformats/hashes/sha2'
 
 /**
  * @param {CID[]} roots
- * @param {AsyncIterable<Block>|Iterable<Block>} blocks
+ * @param {AsyncIterable<Block>|Iterable<Block>|AsyncGenerator<{cid: CID, bytes: Uint8Array}>} blocks
```

Comment:
No longer needed, reverting!

Reply:
I _think_ you could have used `AsyncIterableIterator<Block>` here instead of all those.